.github/scripts/update-package.py

```
@@ -0,0 +1,545 @@
+            print(f"Warning: Low rate limit! Resets at {reset_time}")
+        
+    except requests.RequestException as e:
+        print(f"Could not check rate limit: {e}", file=sys.stderr)
+
+def get_latest_release(owner: str, repo: str) -> Dict[str, Any]:
+    """Get latest release info from GitHub API."""
+    headers = get_github_headers()
+    url = f"https://api.github.com/repos/{owner}/{repo}/releases/latest"
+    
+    try:
+        response = requests.get(url, headers=headers)
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)